src/Pi.cpp

```diff
@@ -789,6 +786,7 @@ void Pi::Quit()
 		_pclose(Pi::ffmpegFile);
 	}
 	Projectile::FreeModel();
+	Beam::FreeModel();
```

Comment:
@fluffyfreak , just because in PR description I forgot to mention this and change in src/LuaShip.cpp#678